cw/cw305/capture.py

```diff
@@ -642,6 +732,17 @@ def sha3_fvsr_key(ctx: typer.Context,
     capture_end(ctx.obj.cfg)
 
 
+@app_capture.command()
+def sha3_fvsr_batch(ctx: typer.Context,
```

Comment:
I think to be consistent, this command should be called `sha3_fvsr_key_batch`. Would you mind changing this?

Other than that, this looks really great. I just tested it on the FPGA. Nice work!

Reply:
Perfect, thanks!